editor/scene/sprite_frames_editor_plugin.cpp

```diff
@@ -679,6 +679,25 @@ void SpriteFramesEditor::_notification(int p_what) {
 		case NOTIFICATION_LAYOUT_DIRECTION_CHANGED:
 		case NOTIFICATION_TRANSLATION_CHANGED: {
 			_update_show_settings();
+
+			anim_speed->set_suffix(TTR("FPS"));
+
+			// Same as in `_update_library_impl()`, but update names only for items without texture.
```

Comment:
If it's the same, the code should be extracted to a new method, to avoid duplicating logic.

Reply:
Inside `_update_library_impl()` `add_item()` is called instead of `set_item_text()`

Reply:
Unfortunately, I haven't figured out how to combine this and extract it into a separate function - the logic partially overlaps, but only partially. That's why I added additional comments. If you have any suggestions on how to do this, I will be glad to do it.

Reply:
Got it, thank you!